src/utils/hooks.jsx

```diff
@@ -22,17 +24,41 @@ export const useRenderContactHelpText = (enterpriseConfig) => {
   return renderContactHelpText;
 };
 
+let cachedApiKey = null;
+
+export const useAlgoliaSearchApiKey = (config) => {
```

Comment:
[inform] We've been starting to use `@tanstack/react-query`'s `useQuery` hook to make API `GET` requests moving forward, as it removes a fair amount of boilerplate (e.g., loading states, `useEffect`, etc.), the need for custom client-side caching support, etc.

Reply:
Thank for letting me know, @adamstankiewicz. Would you like me to re-write this using `useQuery`, or it's fine as is?